modules/rostests/apitests/user32/LoadImage.c

```diff
@@ -45,6 +57,26 @@ static void test_LoadImage_DataFile(void)
     }
 }
 
+static void test_LoadImage_System(void)
+{
+    HINSTANCE hInst = GetModuleHandleW(L"USER32");
+    static const WORD icomap[][2] = {
+        { 100, (WORD)(SIZE_T)IDI_APPLICATION },
+        { 101, (WORD)(SIZE_T)IDI_WARNING },
+        { 102, (WORD)(SIZE_T)IDI_QUESTION },
+        { 103, (WORD)(SIZE_T)IDI_ERROR },
+        { 104, (WORD)(SIZE_T)IDI_INFORMATION },
+        { 105, (WORD)(SIZE_T)IDI_WINLOGO }
+    };
+
+    for (UINT i = 0; i < _countof(icomap); i++)
+    {
+        HICON hIcoRes = LoadIconW(hInst, MAKEINTRESOURCEW(icomap[i][0]));
+        HICON hIcoSys = LoadIconW(NULL, MAKEINTRESOURCEW(icomap[i][1]));
+        ok(hIcoRes && AreIconsEqual(hIcoRes, hIcoSys), "SysIcon %d must be resource %d\n", icomap[i][1], icomap[i][0]);
```

Comment:
```suggestion
        ok(AreIconsEqual(hIcoRes, hIcoSys), "SysIcon %d must be resource %d\n", icomap[i][1], icomap[i][0]);
```
as `SHAreIconsEqual()` checks `NULL` values.

Reply:
Guaranteed? ...on Windows? This code also needs to run on Windows. This is just a test, no need to optimize.